packages/tablev2/style.ts

```diff
@@ -0,0 +1,85 @@
+import * as dt from "../design-tokens/build/js/designTokens";
+import { css, cx } from "emotion";
+import { pseudoElTriangle } from "../shared/styles/styleUtils";
+
+export const headerCss = css`
+  box-sizing: border-box;
+  border-top: 1px solid ${dt.themeBorderHeavy};
+  border-bottom: 1px solid ${dt.themeBorderHeavy};
+`;
+
+export const table = (
+  gridTemplateFragments: string[],
+  firstColSticky: boolean = false
+) => css`
+  display: grid;
+  grid-template-columns: ${gridTemplateFragments.join(" ")};
+  position: relative;
+  overflow: scroll;
+  ${firstColSticky && stickyFirstCol(gridTemplateFragments.length)}
+`;
+
+const stickyFirstCol = (n: number) => `
+  .${th}:first-child, .${cell}:nth-child(${n}n + 1) {
```

Comment:
Newer versions of emotion recommend against interpolating class names: https://github.com/emotion-js/emotion/blob/ca95f385f7ce3da6d53de1a652b3b219f11434c4/packages/serialize/src/index.js#L248.

Reply:
That being said, I really love this use of the `:nth-child` selector 😍 .

Reply:
thanks for fixing! 💜